composer/algorithms/swa/swa.py

```diff
@@ -41,43 +42,79 @@ class SWA(Algorithm):
             from composer.algorithms import SWA
             from composer.trainer import Trainer
             swa_algorithm = SWA(
-                swa_start=0.8
+                swa_start=0.6,
+                swa_end=0.8
             )
             trainer = Trainer(
                 model=model,
                 train_dataloader=train_dataloader,
                 eval_dataloader=eval_dataloader,
-                max_duration="1ep",
+                max_duration="10ep",
                 algorithms=[swa_algorithm],
                 optimizers=[optimizer]
             )
 
     Args:
-        swa_start (float): Fraction of training completed before stochastic weight
-            averaging is applied. Defalt = ``0.8``.
+        swa_start (float, optional): Fraction of training completed before stochastic
```

Comment:
Should these be time strings instead of floats?

Reply:
Wooo, go time abstraction!